doc/engine/installing/linux-generic.md

```diff
@@ -0,0 +1,49 @@
+# Installing the OpenQuake Engine on a generic Linux distrbution
+
+The OpenQuake Engine is also available in the form of **self-installable binary distribution**.
+
```

Comment:
Yes, why not?
The installer already has an option to work in an unmanned environment (`./openquake-installer-linux-2.0.0.run -- -d $HOME`)